pom.xml

```diff
@@ -155,6 +155,12 @@
         <artifactId>commons-io</artifactId>
         <version>2.11.0</version>
       </dependency>
+      <dependency>
+        <!-- Temp work-around until httpclient upgrades its codec version -->
```

Comment:
this comment is a bit misleading. last release from 4.5.x series of http client was in 2020